exporters/failure/collector_jira.py

```diff
@@ -103,10 +103,16 @@ def _connect_to_jira(self) -> JIRA:
         """Connect to JIRA instance which may be cloud based or self-hosted."""
         try:
             # Connect to JIRA
-            jira_client = JIRA(
-                options={"server": self.tracker_api},
-                basic_auth=(self.username, self.token),
-            )
+            if not self.username:
+                jira_client = JIRA(
+                    options={"server": self.tracker_api},
+                    token_auth=(self.token),
```

Comment:
I would remove the `()` here. I thought this was a tuple, but it is just a string